tests/test_object.py

```diff
@@ -93,6 +93,30 @@ def test_error_handling(self):
         except RuntimeError as e:
             self.assertEqual(e.args[0].count("Error"), 1)
 
+    def test_equality(self):
+        """Test __eq__ and __ne__
+        """
+        # pick a case where we know there is a difference between
+        # attributes that are unset and those set to the default value
+        frame1 = astshim.Frame(2)
+        frameSet1 = astshim.FrameSet(frame1)  # base is unset
+        self.assertFalse(frameSet1.test("Base"))
+        frameSet2 = astshim.FrameSet(frame1)
+        frameSet2.base = 1
+        self.assertTrue(frameSet2.test("Base"))
+        self.assertTrue(frameSet1 == frameSet2)
+        self.assertFalse(frameSet1 != frameSet2)
+        self.assertEqual(frameSet1, frameSet2)
+
+        self.assertNotEqual(frameSet1, frameSet1.getInverse())
+        self.assertEqual(frameSet1, frameSet1.getInverse().getInverse())
+        self.assertEqual(frameSet1.getInverse(), frameSet2.getInverse())
+
+        frame3 = astshim.Frame(2)
+        frame3.title = "Frame 3"
+        frameSet3 = astshim.FrameSet(frame3)
+        self.assertNotEqual(frameSet1, frameSet3)
```

Comment:
I added a test for two objects with equal attributes. Good suggestion!

I don't think testing frameSet1 against itself is interesting because the fallback is pointer equality.

The attribute containing a hash is a good point, but no longer relevant because I am no longer stripping hash marks.